src/Parser.js

```diff
@@ -46,6 +46,7 @@ class Parser {
     for (let ext in extensions) {
       this.registerExtension(ext, extensions[ext]);
     }
+    this.isProdBuild = options.isProduction;
```

Comment:
You don't need to set this as it's part of the options?

Reply:
The options don't get stored inside the parser object. Should I do `this.options = options` instead?